frontend_app_admin_workspace_user/src/component/AddUserForm.jsx

```diff
@@ -11,14 +13,27 @@ export class AddUserForm extends React.Component {
 
     this.state = {
       newUserName: '',
+      newUserUsername: '',
       newUserEmail: '',
       newUserPassword: '',
-      newUserProfile: ''
+      newUserProfile: '',
+      popoverEmailInfoOpen: false
     }
   }
 
+  handleTogglePopoverEmailInfo = () => {
+    this.setState(prevState => ({
+      popoverEmailInfoOpen: !prevState.popoverEmailInfoOpen
+    }))
+  }
+
   handleChangeNewUserName = e => this.setState({ newUserName: e.target.value })
 
+  handleChangeNewUserUsername = e => {
+    this.setState({ newUserUsername: e.target.value })
```

Comment:
I'm wandering if there should be some validation for username

If it has been decided to let the backend handle it for the first version, I'm fine with it

Reply:
As validation we have the availability that is given in real time as the user is typing. Also there are errors sent by the backend or flash messages when the user try to confirm (standard behaviour in Tracim).
Did you have any specific validation in mind? For example, check if the username characters are allowed in real time too?

Reply:
No I thought that by putting the .target.value immediately in the state meant that there wasn't any validation on it.
If these is some, it's fine

Reply:
you can resolve